data2rdf/pipelines/main.py

```diff
@@ -210,6 +210,15 @@ def graph(cls) -> Graph:
             graph += cls._validate_additional_triples(cls.additional_triples)
         return graph
 
+    def to_dict(self, schema: Callable = None) -> "List[Dict[str, Any]]":
+        """Return list of general metadata as DSMS custom properties"""
+        if self.mode == PipelineMode.ABOX:
+            return self.parser.abox.to_dict(schema=schema)
+        else:
+            raise NotImplementedError(
+                "`to_dict()` is not available in `tbox`-mode."
+            )
+
     @property
     def plain_metadata(cls) -> Dict[str, Any]:
```

Comment:
Note that `cls` is being used here, as well as in other places, whereas `self` would be the correct term.